<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthWindow</class>
 <widget class="QWidget" name="AuthWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #808080; 
color: black;
padding: 10px; 
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-left-radius: 15px;
border-bottom-right-radius: 15px;</string>
   </property>
   <property name="text">
    <string>RoomSched</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #000000; 
    border-radius: 15px;
    padding: 5px;
    background: white;
	color: black;
}

QLineEdit:focus {
    border: 2px solid #2F4F4F; 
	color: black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Введите ФИО...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mailInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #000000; 
    border-radius: 15px;
    padding: 5px;
    background: white;
	color: black;
}

QLineEdit:focus {
    border: 2px solid #2F4F4F; 
	color: black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Введите email...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneInput">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #000000; 
    border-radius: 15px;
    padding: 5px;
    background: white;
	color: black;
}

QLineEdit:focus {
    border: 2px solid #2F4F4F; 
	color: black;
}</string>
   </property>
   <property name="inputMask">
    <string>+7 (999) 999-99-99;_</string>
   </property>
   <property name="text">
    <string>+7 () --</string>
   </property>
   <property name="placeholderText">
    <string>Введите телефон...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
padding: 5px;
background: #b0b0b0;
color: black;</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <zorder>nameInput</zorder>
  <zorder>label</zorder>
  <zorder>mailInput</zorder>
  <zorder>phoneInput</zorder>
  <zorder>loginButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
